pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[project]
+name = "expenses-manager"
+version = "0.1.0"
+description = "Expense Manager API using FastAPI and SQLAlchemy"
+readme = "README.md"
+requires-python = ">=3.10"
+
+authors = [
+    { name = "Piotr Gołębiewski", email = "pgolebiewski07@gmail.com" }
+]
+
+dependencies = [
+    "fastapi==0.115.6",
+    "uvicorn==0.34.0",
+    "SQLAlchemy==2.0.44",
+    "pydantic==2.10.6",
+    "pydantic_core==2.27.2",
+    "matplotlib==3.10.7",
+]
+
+[project.optional-dependencies]
+test = [
+    "pytest>=7.0",
+    "httpx>=0.27",
+]
+
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+# --------------------------
+# PYTEST CONFIG (minimalne)
+# --------------------------
+[tool.pytest.ini_options]
+pythonpath = ["."]
```

Comment:
I think this one is too much, is not needed. In real development you would like to run test files in separation to speedup feedback loop or fix file by file, not all at once.